vertical-pod-autoscaler/pkg/recommender-external/logic/recommender.go

```diff
@@ -0,0 +1,62 @@
+package logic
+
+import (
+	"k8s.io/klog/v2"
+
+	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender-external/model"
+	other_logic "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/logic"
+	other_model "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
+)
+
+// PodResourceRecommender computes resource recommendation for a Vpa object.
+type PodResourceRecommender interface {
```

Comment:
I think we can use the existing interface:
In fact the recommender is composing the PodResourceRecommender, but in fact it is a stateless object that is consumed only in one place inside the code [here](https://github.com/DataDog/autoscaler/pull/73/files#diff-9a40053a0fcd9eb3b2ec95cf365aa74e250580017bf464626054dd34ff717076R108)

In fact you could construct the PodResourceRecommender at that time and pass it the `model.ContainerNameToRecommendation` for composition, and then you call the method that has the same signature than the upstream interface and that used the composed `model.ContainerNameToRecommendation`

Reply:
Ah good idea, doing this

Reply:
done